.delivery/config.json

```diff
@@ -0,0 +1,33 @@
+{
+  "version": "2",
+  "build_cookbook": {
+    "name": "build_cookbook",
+    "path": ".delivery/build_cookbook"
+  },
+  "skip_phases": [
```

Comment:
[nit] It's probably just me, but I find existing empty recipes for skipped phases a bit distracting. (Not sure if it is actually somehow required to have those stubs...)